lib/embulk/guess/csv.rb

```diff
@@ -104,6 +104,11 @@ def guess_lines(config, sample_lines)
 
         sample_records = split_lines(parser_guessed, true, sample_lines, delim, {})
 
+        # It should fail if CSV parser cannot parse sample_lines.
+        if sample_records.nil? || sample_records.empty?
```

Comment:
What happens when an empty file (or a header-only file) is given as CSV?

I thought it can easily happen when users are creating multiple CSVs time-by-time.